### PR TITLE
Implementation and Readme Changes

### PR DESCRIPTION
**Pull request by Shama-K**
This PR is created for following changes:

- Updated Readme files and PowerShell Scripts.
- Removed projects from [3.-Web-api-call-Microsoft-graph-for-personal-accounts](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/tree/shama-k/Master+MIW2.0+ReadMePoP/3.-Web-api-call-Microsoft-graph-for-personal-accounts) and with minor updates in [2. Web API now calls Microsoft Graph](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/tree/shama-k/Master%2BMIW2.0%2BReadMePoP/2.%20Web%20API%20now%20calls%20Microsoft%20Graph), the solution can be used to test the scenario. Readme file contains the required updates.
- Replaced the Console client app to WPF app in [4.-Console-app-calls-web-API-with-PoP](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/tree/shama-k/Master%2BMIW2.0%2BReadMePoP/4.-Console-app-calls-web-API-with-PoP) .
- Merged PR [update to microsoft identity web 0 2 0](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/pull/148).

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ x] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

Use manual steps or app creation scripts to configure the sample, and run the sample.
Test the code.

### CODE REVIEW

**Comment by jennyf19** [comment]
@Shama-K we merged the API changes, looks like there are some merge conflicts. Let us know if you need help.

File: 2. Web API now calls Microsoft Graph/README-incremental.md [hunk 1]
```diff
@@ -15,173 +15,118 @@ endpoint: Microsoft identity platform
...
 
-### Step 4: Run the sample
+### Step 3: Run the sample
 
 Clean the solution, rebuild the solution, and run it
 
 ### Alternative architecture
 
-This part of the sample uses different client ID for the client and the service and uses the on-behalf-of flow. If you are the author of both the client and the service, you might alternatively want to use the same client ID in the Client and the Service. This approach is described in the third part of the tutorial [3.-Web-api-call-Microsoft-graph-for-personal-accounts](../3.-Web-api-call-Microsoft-graph-for-personal-accounts)
+This part of the sample uses different client ID for the client and the service. If both the client and the service are part of same application topology, you might want to use the same client ID in the Client and the Service. This approach is described in the third part of the tutorial [3.-Web-api-call-Microsoft-graph-for-personal-accounts](../3.-Web-api-call-Microsoft-graph-for-personal-accounts/README-incremental.md)
```
Comment (jmprieur) [inline 2. Web API now calls Microsoft Graph/README-incremental.md:111]: Do we still have the chapter 3 ?

File: 2. Web API now calls Microsoft Graph/README-incremental.md [hunk 2]
```diff
@@ -15,173 +15,118 @@ endpoint: Microsoft identity platform
...
 Update `Startup.cs` file:
 
-- Add a using for `Microsoft.Identity.Client`
-
 - In the `ConfigureServices` method, replace:
 
   ```CSharp
-  services.AddAuthentication(AzureADDefaults.BearerAuthenticationScheme)
-          .AddAzureADBearer(options => Configuration.Bind("AzureAd", options));
+  services.AddProtectedWebApi(Configuration);
```
Comment (jmprieur) [inline 2. Web API now calls Microsoft Graph/README-incremental.md:128]:
```suggestion
  services.AddMicrosoftWebApiAuthentication(Configuration);
```

File: 4.-Console-app-calls-web-API-with-PoP/TodoListClient/MainWindow.xaml.cs [hunk 3]
```diff
@@ -286,9 +224,12 @@ private async void AddTodoItem(object sender, RoutedEventArgs e)
                 return;
             }
 
+            //
             // Call the To Do service.
+            //
+
             // Once the token has been returned by MSAL, add it to the http authorization header, before making the call to access the To Do service.
-            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", result.AccessToken);
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("PoP", result.AccessToken);
```
Comment (jmprieur) [inline 4.-Console-app-calls-web-API-with-PoP/TodoListClient/MainWindow.xaml.cs:232]: This is different with Pop.
we pass the request, and therefore this is not needed.

Do we really want to replace the console app by the WPF app?
this is way more complicated for Pop?

cc: @kalyankrishna1 what do you think?

Comment (Shama-K) [reply 4.-Console-app-calls-web-API-with-PoP/TodoListClient/MainWindow.xaml.cs]: Changed the client application to console as suggested.
Thank you.

File: 2. Web API now calls Microsoft Graph/README-incremental.md (hunk 2 above)
Comment (Shama-K) [reply 2. Web API now calls Microsoft Graph/README-incremental.md]: Updated the Readme.
Thank you.

**Pull request merged**